TombEngine/Game/Lara/lara_one_gun.cpp

```diff
@@ -185,7 +190,8 @@ void AnimateShotgun(ItemInfo& laraItem, LaraWeaponType weaponType)
 						break;
 
 					case LaraWeaponType::HarpoonGun:
-						FireHarpoon(laraItem);
+					
+						FireHarpoon(laraItem, nullptr);
```

Comment:
If you add new argument for the function which is mostly always passed as `nullptr`, you can make this argument default in declaration (.h file).